pkg/api/jobartifacts/apitypes.go

```diff
@@ -0,0 +1,30 @@
+package jobartifacts
+
+type QueryResponse struct {
+	Errors  []JobRunError `json:"errors,omitempty"`
+	JobRuns []JobRun      `json:"jobRuns,omitempty"`
+}
+
+type JobRun struct {
+	// ID is string because some parsers translate long ints into scientific notation
+	ID      string `json:"id"`
+	URL     string `json:"url"`
+	JobName string `json:"jobName"`
+	// NOTE: limited per maxJobFilesToScan, sets Truncated if more files match
+	Artifacts             []JobRunArtifact `json:"artifacts"`
+	ArtifactListTruncated bool             `json:"artifactListTruncated"`
```

Comment:
I don't think we've written this down anywhere, but Sippy tends to use snake case for the API's, e.g. component readiness responses in `pkg/apis/api/componentreport`.

Again, very inconsistent but many API response types are in `pkg/apis`.